Makefile

```diff
@@ -46,6 +46,10 @@ $(FONT_AWESOME_TARGET): ## Downloads the Docsy Font Awesome dependency.
 gen-content: ## Generates content from external sources.
 	hack/gen-content.sh
 	hack/import-flux2-assets.sh
+	command -v yq > /dev/null || test -x ./yq || \
+		curl -Lfs https://github.com/mikefarah/yq/releases/download/v4.7.1/yq_linux_amd64 -o yq && \
+		chmod a+x yq
```

Comment:
This is not very nice. Can anyone help find a solution which works on the Netlify builds? (Ubuntu AFAIK.)

Reply:
I see no issues with this, we do the same thing with GH Actions: https://github.com/fluxcd/flux2-kustomize-helm-example/blob/main/.github/actions/tools/entrypoint.sh